addons/website_slides/models/slide_channel.py

```diff
@@ -20,6 +20,7 @@ class ChannelUsersRelation(models.Model):
     _description = 'Channel / Partners (Members)'
     _table = 'slide_channel_partner'
 
+    active = fields.Boolean(default=True)
```

Comment:
Asking to have your thoughts
1/ what about slide_channel_ids field on res.partner ? Should we display only active memberships, or deactivated one for reporting purpose ?
2/ what about channel_partner_ids field on slide.channel ? Actually, same question :D
3/ what about partner_ids field on slide.channel ? as it is a m2m using the slide_channel_partner table, it will not take into account the active flag. I think it should probably be matching channel_partner_ids semantic (active only, or all -> partner_ids should do the same).

In a more general way, question is: what do we want to display in backend ? Active memberships, or all memberships with their flag and action buttons ?